wikimedia/v1/summary_v1.yaml

```diff
@@ -92,9 +94,9 @@ paths:
               uri: /{domain}/sys/action/query
               body:
                 prop: 'extracts|pageimages'
-                redirects: ''
+                redirects: 'true'
```

Comment:
Yeah, good point. Was changing this back & forth while hunting down the mysterious missing thumbnail issue, and ended up on the string. Will change it to a boolean.